src/routes/Account/modules/account.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const REQUEST_ACCOUNTS = 'REQUEST_ACCOUNTS'
```

Comment:
I think usually people format constants for actions like `REQUEST_ACCOUNTS, REQUEST_ACCOUNTS_SUCCESS, REQUEST_ACCOUNTS_FAILURE`; first indicates a request is inflight, second is a success handler, third is a failure handler.  not quite sure whether this is commonly accepted or just one person's way of doing things though, but seems to make sense to me